content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -155,7 +155,8 @@ different Kubernetes components.
 | `NetworkPolicyEndPort` | `true` | Beta | 1.22 |  |
 | `NodeSwap` | `false` | Alpha | 1.22 | |
 | `NodeOutOfServiceVolumeDetach` | `false` | Alpha | 1.24 | |
-| `OpenAPIEnums` | `false` | Alpha | 1.23 | |
+| `OpenAPIEnums` | `false` | Alpha | 1.23 | 1.23 |
+| `OpenAPIEnums` | `false` | Beta | 1.24 | |
```

Comment:
I think it's enabled by default, if I'm correct. @jiahuif ?

Reply:
Correct. That's a problem. Sending a fix.